resources/config/config.yaml

```diff
@@ -14,6 +14,10 @@ params:
     other: "--minimum-length 1 --nextseq-trim 20"
   multiqc: ''
   bowtie2: ''
+  sourmash:
+    k: 31
```

Comment:
I think this may run into trouble if you want to profile with multiple values of `k`. You could consider having all of the parameters to sourmash be designated in the config file as a string, like `'k=31,k=51,scaled=1000'`

Reply:
Oh, also if you have two different sets of .sig files made with different values of `k`, that will create issues generating the distance matrix with `compare`, since then you have to specify which values of `k` you want to use.

Reply:
Oh, I just noticed that you can add more parameters in the `extra` section, so maybe this is fine.

Reply:
You can. It would be possible to specify K as a wildcard for this purpose. For our use-case though, I wonder if it's worth it?

Reply:
Yeah, I agree if we want to keep a database of all samples, having one universal value for k would simplify things. Based on the [FAQ](https://sourmash.readthedocs.io/en/latest/using-sourmash-a-guide.html?highlight=51#what-k-mer-size-s-should-i-use):
>We suggest including k=31 and k=51. k=51 gives you the most stringent matches, and has very few false positives. k=31 may be more sensitive at the genus level.

I wonder if k=51 might be a better value to stick with, mainly because its higher stringency appears to be more in line with our goal of identifying species/strain-level similarities between samples?

Reply:
I think for the purposes of prototype selection we actually want the more general / more sensitive parameters. I don't think the minhash approach for *positive strain ID* really is very useful in our case: we won't be trawling through petabytes of data to find samples where a particular genome occurs. We'll be doing rough passes to find similar samples, followed by read mapping approaches with a much higher level of specificity for strain-level analysis. Put another way, a more stringent kmer will be more likely to give saturated dissimilarity measures among samples, making the distance matrix less useful for prototype selection.